翻译v3.4自定义构建纹理压缩处理 (#1934)
cocos-creator/creator-docs: en/editor/extension/extension-manager.md

# Extension Manager Panel Description

> The **Extension Manager** is used to manage extensions in the editor. Click on **Extension -> Extension Manager** in the top menu bar of Creator to open.

Reviewer: ```suggestion
The **Extension Manager** is used to manage extensions in the editor and can be opened by clicking on **Extension -> Extension Manager** in the top menu bar of the editor.
```

## Extension List

The **Extension Manager** panel displays the corresponding extensions according to the currently selected extension type and by filtering through the search bar, etc.

![simple-extension](./image/extension-list.png)

In the extension list, the extension-related information is displayed on the left side of the extension, and the descriptions of the buttons on the right side are as follows:

- ![folder](first/folder.png): Opens the extension package in the operating system's file manager.

- ![refresh](first/refresh.png): Refresh button, used to reload the current extension.

- ![delete](first/delete.png): Uninstall the extension, when clicked it will remove the current extension from the extension list and delete the extension package.

- ![enable](first/enable.png): Enable/disable the extension, the button shows green for enable and gray for disable.

## Extension Types

Creator currently classifies extensions as **Internal**, **Project**, **Global**, and **Develop**, click to switch and display the list of extensions of the current type.

### Internal Extensions

> The extensions in the Internal extensions tab are all built-in extensions of the editor. The internal extensions cannot be closed, uninstalled or refreshed, and external extensions cannot be imported as internal extensions, so the Scan and Import buttons above are grayed out and not available.

Reviewer: ```suggestion
The extensions in the Internal extensions tab are all built-in extensions of the editor. The internal extensions cannot be closed, uninstalled or refreshed, and external extensions cannot be imported as internal extensions. The Scan and Import buttons above are grayed out and not available.
```

| :--------------- | :---------- |
| Extension List | Shows extensions installed in the project extension directory. |
| Import Button | Imports the ZIP package of the extension and extract the ZIP to the project extension directory. |

### Global Extensions

The extensions in the Global Extensions tab are available to all projects that have the extension enabled. The global extensions are located in the following directories:

- **Windows**: `%USERPROFILE%\.CocosCreator\extensions`
- **MacOS**: `$HOME/.CocosCreator/extensions`

![simple-extension](./image/extension-global.png)

| Function | Description |
| :--------- | :----------- |
| Extension List | Shows the extensions installed in the global extension directory. |
| Import Button | Imports the ZIP package of the extension and extract the ZIP to the global extension directory. |

### Develop

To facilitate the development of extensions, the **Develop** tab allows **temporary import** of extensions (excluding **current project extensions** and **global extensions**).

| Function | Description |
| :------ | :-------- |
| Extension List | Show temporarily installed extensions. |
| Scan Button | This button is disabled. |

> | Import Button | Click this button to **temporarily** register the specified extensions to the extension list in the **Development** tab, but will not copy the corresponding extension packages to the current project. Note that temporarily registered extensions are not currently recorded and will need to be re-registered the next time the project is opened. |

Reviewer 2: ```suggestion
| Import Button | Click this button to **temporarily** register the specified extensions to the extension list in the **Develop** tab, but will not copy the corresponding extension packages to the current project. Note that temporarily registered extensions are not currently recorded and will need to be re-registered the next time the project is opened. |
```

> > **Note**: if you click the Uninstall button of an extension in the **Develop** tab, it will only remove the extension in the extension list, and will not delete the extension folder.

Reviewer: ```suggestion
> **Note**: clicking the Uninstall button of an extension in the **Develop** tab, will only remove the extension in the extension list, and will not delete the extension's folder.
```

> > **Note**: clicking the Uninstall button of an extension in the **Develop** tab, will only remove the extension in the extension list, and will not delete the extension's folder.

Reviewer 2: ```suggestion
> **Note**: clicking the Uninstall button of an extension in the **Develop** tab, will only remove the extension in the extension list, and will not delete the extension package.
```
